virtcontainers/cgroups.go

```diff
@@ -100,9 +100,15 @@ func (s *Sandbox) applyCPUCgroup(rc *specs.LinuxResources) error {
 	// when new container joins, new CPU could be hotplugged, so we
 	// have to query fresh vcpu info from hypervisor for every time.
 	tids, err := s.hypervisor.getThreadIDs()
-	if err != nil || tids == nil {
+	if err != nil {
 		return fmt.Errorf("failed to get thread ids from hypervisor: %v", err)
 	}
+	if tids == nil {
+		// If there's no tid returned from the hypervisor, this is not
+		// a bug. It simply means there is nothing to constraint, hence
```

Comment:
s/constraint/constrain

Looks good otherwise (do same change in commit message)

Reply:
ok!

Reply:
updated!